requirements.txt

```diff
@@ -6,4 +6,4 @@ MarkupSafe==1.1.1
 protobuf==3.12.2
 pypandoc==1.5
 PyYAML==5.3.1
-dataclasses==0.7
\ No newline at end of file
+dataclasses==0.6; python_version < '3.7'
```

Comment:
LGTM, assuming this syntax is understood by bazel python rules properly (the python_version < '3.7' part). Note, requirements.txt is currently used only by bazel.

Reply:
apparently it does not :(  (or something else is wrong)